cypress/e2e/changeAssetOwnership.feature

```diff
@@ -1,7 +1,6 @@
+@asset-ownership
```

Comment:
I think this is `propety` MFE view. If we're tagging by MFE name, this needs changed.